src/modules/i18n/locales/de.json

```diff
@@ -653,5 +653,9 @@
     "kml_gpx_file_empty": "Die KML/GPX-Datei ist leer",
     "large_size": "Gross",
     "extra_large_size": "Extra Gross",
-    "duplicate_layer": "Karte duplizieren"
+    "duplicate_layer": "Karte duplizieren",
+    "feedback_empty_warning": "Die Berichtsnachricht darf nicht leer sein",
+    "operation_successful": "Erfolg! ",
```

Comment:
Is the extra whitespace at the end intentional? Or might it break some formatting?

Reply:
I believe this might be a whitespace that was unseen in the google sheets.